pkg/crc/machine/generate_bundle.go

```diff
@@ -33,17 +32,21 @@ func (client *client) GenerateBundle(forceStop bool) error {
 	}
 
 	// Stop the cluster
-	currentState, err := client.Stop()
-	if err != nil {
+	if _, err := client.Stop(); err != nil {
 		if forceStop {
 			if err := client.PowerOff(); err != nil {
 				return err
 			}
+		} else {
+			return err
 		}
-		return err
```

Comment:
The commit logs says ` Fix the state of VM when generating the bundle ` but it seems this is also fixing  the fact that we'd always error out when forced shutdown was triggered, even if it was successful?

Reply:
@cfergeau yes fixed it, created new commit to handle it.